app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,22 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
+    belongs_to :appliance,
+               class_name: 'Atmosphere::Appliance'
+
+    has_many :action_logs, -> { order(id: :asc) },
+             dependent: :destroy,
```

Comment:
I was not clear. After appliance is destroyed by the user it can been flagged as destroyed, but still present in DB for further, eventually diagnostics. No automatic active appliance destroying :smile: